src/ovms.h

```diff
@@ -79,6 +79,14 @@ typedef enum OVMS_BufferType_enum {
     OVMS_BUFFERTYPE_HDDL,
 } OVMS_BufferType;
 
+typedef enum OVMS_ServableState_enum {
```

Comment:
The only thing we are really missing is information when servable is in loading state that is it temporary (it is in process of loading), or is it due to broken configuration which require admin attention. We should add state for that.

I think we shouldn't expose via C-API details why servable failed to load. In case of TFS it is only info (UNKNOWN/PRECONDITION_FALIED).